types/dockerode/index.d.ts

```diff
@@ -1428,19 +1473,9 @@ declare namespace Dockerode {
         abortSignal?: AbortSignal;
     }
 
-    interface ServiceCreateResponse {
-        ID: string;
-        Warnings?: string[] | undefined;
-    }
-
     interface ServiceListOptions {
-        filters: {
-            id?: string[] | undefined;
-            label?: string[] | undefined;
-            mode?: Array<'replicated' | 'global'> | undefined;
-            name?: string[] | undefined;
-        };
-
+        filters?: string | undefined;
```

Comment:
this version break working project, dockerode accept both declaration, this should be 
 ```
 filters?: string | {
            id?: string[] | undefined;
            label?: string[] | undefined;
            mode?: Array<'replicated' | 'global'> | undefined;
            name?: string[] | undefined;
        } | undefined;
```

Reply:
Indeed, thank you ! Here is the fix: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63263

Reply:
> fix has been published in [version 3.3.14](https://www.npmjs.com/package/@types/dockerode/v/3.3.14)

Thank you ! It's good now